### PR TITLE
Adding en.yml for another package, a customized rep finder

### PR DESCRIPTION
**Pull request by bright-cloud-studio**
I am adding the folder and en.yml file for another package we have, a rep finder.

### CODE REVIEW

File: meta/bright-cloud-studio/rep-finder/en.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+en:
+    title: Rep Finder
+    description: >
+        Adds a frontend module to search for representatives by ZIP Code, and a backend system for managing it.
```
Comment (xchs) [inline meta/bright-cloud-studio/rep-finder/en.yml:4]:
```suggestion
        Adds a front end module to search for representatives by ZIP Code, and a back end system for managing it.
```

**Comment by bright-cloud-studio** [comment]
Thanks for pointing that out, I was going to go in and change it but when it said the checks had passed I wasn't sure what was going on there for a minute. Front end and back end are both two words, I'll keep that in mind going forward.

File: linter/allowlists/en.txt [hunk 2]
```diff
@@ -88,6 +88,7 @@ Head
 HeaderFooterCode
 helpdesk
 History
+hotspot
```
Comment (fritzmg) [inline linter/allowlists/en.txt:91]: The word `hotspot` has no occurrence in your PR - so you don't need to add this entry to the allow list. Also _hotspot_ is a regular English word and does not need to be put on an allow list for the spell checker.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: linter/allowlists/en.txt (hunk 2 above)
Comment (bright-cloud-studio) [reply linter/allowlists/en.txt]: At one point I believe I was using the term "hotspot" and I was getting spellcheck errors. I removed it.

**Review by bright-cloud-studio — COMMENTED** [review]
I've removed "hotspot" from the allowlist.

File: meta/bright-cloud-studio/add-user-fields/en.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+en:
+    title: Add User Fields
+    description: >
+        Adds two custom user fields, image and bio, so they can be displayed on News posts or elsewhere
```
Comment (fritzmg) [inline meta/bright-cloud-studio/add-user-fields/en.yml:4]:
```suggestion
        Adds two custom user fields, image and bio, so they can be displayed on News posts or elsewhere.
```

File: meta/bright-cloud-studio/modal-gallery/en.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+en:
+    title: Modal Gallery
+    description: >
+        A customized gallery with unlimited links that open modal boxes
```
Comment (fritzmg) [inline meta/bright-cloud-studio/modal-gallery/en.yml:4]:
```suggestion
        A customized gallery with unlimited links that open modal boxes.
```

**Review by fritzmg — COMMENTED** [review]
Just some nitpicks for consistency :)

**Review by fritzmg — APPROVED** [review]

**Review by bright-cloud-studio — COMMENTED** [review]
I believe I fixed all these correctly.

**Comment by bright-cloud-studio** [comment]
Github is so confusing at times.

**Review by fritzmg — APPROVED** [review]
I think part of your troubles was that the source branch for your PR was not up to date with the main branch of the upstream repository.

**Pull request merged**